src/core/dfn-panel.runtime.js

```diff
@@ -32,30 +43,50 @@ function dfnPanel() {
       }
       case "hide": {
         hidePanel(panel);
+        panel = null;
         break;
       }
     }
-  });
+  };
+}
+
+/**
+ * @param {MouseEvent|KeyboardEvent} event
+ */
+function deriveCoordinates(event) {
+  if (event instanceof MouseEvent) {
+    return { x: event.clientX, y: event.clientY };
+  }
+  // Otherwise, KeyboardEvent...
+  const target = /** @type HTMLElement */ (event.target);
```

Comment:
Following should work fine also (cleaner IMO):

```suggestion
  /** @type HTMLElement */
  const target = event.target;
```

Reply:
Tried that originally, but doesn't seem to like it.

Reply:
<img width="1048" alt="Screen Shot 2021-04-19 at 1 25 00 pm" src="https://user-images.githubusercontent.com/870154/115177496-b1d5e380-a112-11eb-8e8e-505f29b46ebd.png">